onebusaway-android/src/main/java/org/onebusaway/android/util/UIUtils.java

```diff
@@ -1432,11 +1432,12 @@ public static void closeKeyboard(Context context, View v) {
      * for a stop.  See #700.
      *
      * @param response response from arrivals-and-departures-for-stop API
+     * @param filter   list of routes to not retrieve service alerts for
      * @return a list of all situations (service alerts) that are specific to the stop, routes, and
-     * agency
+     * agency (excluding those specified with filter)
```

Comment:
Same here - I believe the docs are inverted from actual behavior.  Should be:

`agency.  If a route filter list is provided, situations for all stops and agencies are included in the returned list, but only situations scoped for route_ids in the provided filter list are included in the returned list (i.e., situations specified for route_ids that aren't in the filter list are excluded).`